pkg/kubectl/cmd/util/openapi/openapi_getter_test.go

```diff
@@ -18,14 +18,84 @@ package openapi_test
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
 
+	"gopkg.in/yaml.v2"
+
+	"github.com/googleapis/gnostic/OpenAPIv2"
+	"github.com/googleapis/gnostic/compiler"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	"k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi"
 	tst "k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi/testing"
 )
 
+// Test utils
+var data apiData
+
+type apiData struct {
+	sync.Once
+	data *openapi_v2.Document
+	err  error
+}
+
+func (d *apiData) OpenAPISchema() (*openapi_v2.Document, error) {
```

Comment:
Why not reuse your code in https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/openapi/testing/openapi.go#L41?